samples/v1.2/Scenarios/Agenda.json

```diff
@@ -175,7 +179,8 @@
 						{
 							"type": "Image",
 							"horizontalAlignment": "Center",
-							"url": "https://messagecardplayground.azurewebsites.net/assets/CircleGreen_coffee.png"
+							"url": "https://messagecardplayground.azurewebsites.net/assets/CircleGreen_coffee.png",
+							"altText": "Coffee icon"
```

Comment:
>Coffee icon" [](start = 19, length = 12)

Is "icon" redundant here? Alternatively should the other alttext strings include qualifiers like "icon" or "image" too? What's guidance say here?

Reply:
@almedina-ms here's what Sarah from Accessibility team pointed to : https://ux.iu.edu/accessibility/alt-text/ and the Msft training: https://aka.ms/AltTextClass